test/SimpleClientTests/AccountSpec.hs

```diff
@@ -134,8 +140,14 @@ exampleCredentials p = IDTypes.NormalAC (IDTypes.CredentialDeploymentValues
           keys = Map.fromList . zip [0..] $ unsafePerformIO $ replicateM 2 (SS.correspondingVerifyKey <$> SS.newKeyPair SS.Ed25519)
           threshold = 1
           in IDTypes.CredentialPublicKeys keys (IDTypes.SignatureThreshold threshold)
-        (Just regId) = BSH.deserializeBase16 "a1355cd1e5e2f4b712c4302f09f045f194c708e5d0cae3b980f53ae3244fc7357d688d97be251a86735179871f03a46f"
-        (Just share) = BSH.deserializeBase16 "a1355cd1e5e2f4b712c4302f09f045f194c708e5d0cae3b980f53ae3244fc7357d688d97be251a86735179871f03a46fa1355cd1e5e2f4b712c4302f09f045f194c708e5d0cae3b980f53ae3244fc7357d688d97be251a86735179871f03a46f"
+        regIdS = "a1355cd1e5e2f4b712c4302f09f045f194c708e5d0cae3b980f53ae3244fc7357d688d97be251a86735179871f03a46f"
+        shareS = "a1355cd1e5e2f4b712c4302f09f045f194c708e5d0cae3b980f53ae3244fc7357d688d97be251a86735179871f03a46fa1355cd1e5e2f4b712c4302f09f045f194c708e5d0cae3b980f53ae3244fc7357d688d97be251a86735179871f03a46f"
+        (regId, share) = case (BSH.deserializeBase16 regIdS, BSH.deserializeBase16 shareS) of
+          (Just regId', Just shareS') -> (regId', shareS')
+          -- This does not happen since the formats of
+          -- regIds and shareS are base 16 strings and
+          -- hence they are always deserialized.
```

Comment:
Not entirely correct. Base16 is only a part of the check. They were generated correctly that is why it will work.